### PR TITLE
test case to enable BMC feature on smart-proxy

### PR DESCRIPTION
**Pull request by sghai**
Test case to validate BZ:1198015

### CODE REVIEW

File: tests/foreman/cli/test_capsule.py [hunk 1]
```diff
@@ -163,3 +163,30 @@ def test_ssl_capsule(self):
...
+        """@Test: Enable BMC feature on smart-proxy
+
+        @Feature: Capsules
+
+        @Setup: A capsule installed with SSL enabled.
+
+        @Steps:
+
+        1. Enable BMC feature on proxy by running installer with:
+           katello-installer --foreman-proxy-bmc 'true'
```
Comment (Ichimonji10) [inline tests/foreman/cli/test_capsule.py]: You can denote inline code blocks with double backticks, like so:

    1. Enable BMC feature on proxy by running installer with:
        ``katello-installer --foreman-proxy-bmc 'true'``

Comment is not a blocker.

**Comment by Ichimonji10** [comment]
ACK

**Comment by sghai** [comment]
Yeah..done..thanks for review.

**Comment by elyezer** [comment]
Looks good, ACK

@Ichimonji10 feel free to merge if the formatting is OK now.

**Comment by Ichimonji10** [comment]
Thank you @sghai.

**Pull request merged**